misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -58,6 +111,25 @@ internal class RealTransacter(
     }
   }
 
+  private fun isRetryable(e: Exception): Boolean {
+    // TODO(jmuia): check the causal chain.
```

Comment:
I wonder if this is actually necessary

Reply:
I can remove the TODO and if, in practice, we see these exceptions wrapped up we can revisit implementing this check.